lib/src/license_detection/confidence.dart

```diff
@@ -280,8 +280,8 @@ const lesserGplChange = -2;
 final _inducedPhrases = {
   'AGPL': ['affero'],
   'Atmel': ['atmel'],
-  'Apache': ['apache'],
-  'BSD': ['bsd'],
+  // 'Apache': ['apache'],
```

Comment:
At one example I've looked at, it was the URL (apache.org), in another it was the appendix (which included the "Apache License" somewhere). I none of the cases on pub.dev have I seen malicious use of "Apache" in an otherwise 95%+ match for Apache license.

I'd argue that in practice this check should be the inverse: an Apache license SHOULD NOT include text referencing affero, gpl or other license keywords, similarly a GPL license SHOULD NOT reference Apache or BSD....